src/cpp/rtps/DataSharing/DataSharingNotification.hpp

```diff
@@ -93,7 +93,7 @@ class DataSharingNotification
 
 #pragma warning(push)
 #pragma warning(disable:4324)
-    struct alignas (uint64_t) Notification
+    struct alignas (8) Notification
```

Comment:
Maybe using something like `alignas ( sizeof (uint64_t) )` would be cleaner and clearer.